app/content/2022-08-26.md

```diff
@@ -1,45 +1,51 @@
 ---
 title: Weekly Community News - 2022-08-26
 date: 2022-08-26 15:00
-tags: 
+tags: industry, conferences, security
+github: https://github.com/kjaymiller/Python-Community-News/blob/main/app/content/2022-08-26.md
 ---
 
 
-# Heroku ends free tier
+<iframe width="560" height="315" src="https://www.youtube.com/embed/_xyKDQJxNOg" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Not a necessary change today, but we can probably hoist the video id into frontmatter and have the template render the iframe.